packages/canary-web-components/src/components.d.ts

```diff
@@ -315,6 +319,10 @@ export namespace Components {
           * The helper text that will be displayed for additional field guidance. This will default to the text "Use format" along with the `dateFormat` value.
          */
         "helperText"?: string;
+        /**
+          * Hide the visible helper text, it will still be read out by screenreaders.
```

Comment:
Update to follow other boolean prop descriptions e.g. 
```suggestion
          * If `true`, the helper text will be visually hidden, but still read out by screenreaders.
```